corehq/apps/cloudcare/static/cloudcare/js/form_entry/form_ui.js

```diff
@@ -10,7 +10,7 @@ hqDefine("cloudcare/js/form_entry/form_ui", function () {
         ko.bindingHandlers.renderMarkdown = {
             update: function (element, valueAccessor) {
                 var value = ko.unwrap(valueAccessor());
-                value = markdown.render(value || '');
```

Comment:
Are you sure unwrap cannot return null or undefined?

Reply:
The null check is done by [markdown.render](https://github.com/dimagi/commcare-hq/blob/b1a9bef71ab117b24f4166529a255506b0e50b86/corehq/apps/cloudcare/static/cloudcare/js/markdown.js#L152) 
```
md.render(DOMPurify.sanitize(text || "").replaceAll("&#10;", "\n"));
```